backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -51,4 +58,33 @@ public UsersController(WebClientService webClientService) {
         String path = usersPath + "/" + id;
         return webClientService.get(path, null);
     }
+
+    @PostMapping(value = "/users")
```

Comment:
Any particular reason you went with `value = ` for @PostMapping here, unlike all the @GetMappings? I'm fine with either, I just like uniformity.